### DISCUSSION TITLE
Shop Vat Number Field on BO

### DISCUSSION THREAD

**Discussion by cbatista8a** [PrestaShop/PrestaShop#40210] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Currently, from the administration panel (BO) there is no field to indicate the fiscal identification (vat_number), which would be useful in various situations, for example, on invoices, especially with the new European obligation on electronic invoicing.

### Describe the solution you'd like

Include the "vat_number" field in the "AdminStores" controller and return it at least in the 'Shop::getAddress()' class method to have centralized access to this field, because the Address class is the one that has this field implemented but it is only being used in the front for the customers.
[BO screenshot](https://drive.google.com/file/d/1F-CdHJW5z18JjkoR-3SPp6moVbZ-RHIy/view?usp=share_link)
[Code screenshot](https://drive.google.com/file/d/1E8svHmMxZrXfG2aT4ruGSAitJefklD6g/view?usp=share_link)

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by AureRita** [comment]
Hi @cbatista8a  ,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

You can also put your feature [here ](https://build.prestashop-project.org/news/2023/github-discussions-are-now-enabled-in-the-prestashop-project/)

Thank you

**Comment by Hlavtox** [comment]
A nice feature, someone can make a PR for it. It's easy.
